src/openseadragon.js

```diff
@@ -1102,12 +1102,17 @@ function OpenSeadragon( options ){
                 // Extend the base object
                 for ( name in options ) {
                     var descriptor = Object.getOwnPropertyDescriptor(options, name);
-                    if (descriptor.get || descriptor.set) {
-                        Object.defineProperty(target, name, descriptor);
-                        continue;
-                    }
 
-                    copy = descriptor.value;
+                    if (descriptor !== undefined) {
+                        if (descriptor.get || descriptor.set) {
+                            Object.defineProperty(target, name, descriptor);
+                            continue;
+                        }
+
+                        copy = descriptor.value;
+                    } else {
+                        $.console.warn('Undefined descriptior obtained for the "' + name + '" property in extended object.');
```

Comment:
I'm thinking we don't want to extend with inherited properties, so we should do a `continue` in here. As for the warning, it might be useful (and I don't expect it to run in normal circumstances), but the phrasing is something about how there's an inherited property that we're not copying.

Reply:
okay, i've updated the warning message and added a `continue` statement.